classes/items/itembase.hxx

```diff
@@ -43,9 +49,10 @@ public:
     unsigned int get_value() const; //allows for other files to use the private value variable
     unsigned int get_tier() const; // allows for other files to access the private tier variable
     unsigned int get_level() const; //allows for other files to access the level variable
+    unsigned short get_id() const; //allows for other files to access the id variable
 
     //public functions
-    std::string toString() const;
+    std::string Item::toString() const;
```

Comment:
what are you doing here?

Reply:
This is where I was trying to use a giant to string function not sure its working properly this is why I tested by hand everything as far as making the items works

Reply:
The current issue is that you're scoping into the Item class inside of teh Item class. That's not allowed. Hence it won't compile.